fill/sensors/sensor.cc

```diff
@@ -14,47 +12,53 @@ Sensor::Sensor(){
     adc2->setGain(GAIN_ONE);
 
     ADC_vector.push_back(adc2);
+
+    spdlog::info("PT: ADCs initialized");
 }
 
 float Sensor::Read(uint8_t channel, uint8_t index){
     uint16_t reading; 
     float voltage;
 
-    spdlog::info("Reading from ADC {} on channel {}\n", index + 1, channel);
-
     reading = ADC_vector[index]->readADC_SingleEnded(channel);
     voltage = static_cast<float>(reading);
 
-    spdlog::info("Voltage read is {}\n", voltage);
     return voltage;
 }
 
-float Sensor::ReadPT1(){
+float Sensor::ReadPT1() {
     float voltage = Read(PT1_CHANNEL, ADC1_INDEX);
+    spdlog::info("PT1: Voltage read: {}", voltage);
 
-    return 1000*voltage/1349; 
+    return 1000 * voltage / 1349; 
 }
 
-float Sensor::ReadPT2(){
+float Sensor::ReadPT2() {
     float voltage = Read(PT2_CHANNEL, ADC1_INDEX);
+    spdlog::info("PT2: Voltage read: {}", voltage);
 
-    return 1000*voltage/1338;
+    return 1000 * voltage / 1338;
 }
 
-float Sensor::ReadLoadCell(){
+float Sensor::ReadLoadCell() {
     float voltage = Read(LC_CHANNEL, ADC1_INDEX);
+    spdlog::info("LC1: Voltage read: {}", voltage);
 
-    return 250*voltage/1881; 
+    return 250 * voltage / 1881; 
 }
 
-float Sensor::ReadIgnitorOneContinuity(){
+float Sensor::ReadIgnitorOneContinuity() {
     // voltage < 10 implies no continuity, and >= 10 is continuity
     float voltage = Read(IGN_1_CONT_CHANNEL, ADC2_INDEX);
+    spdlog::info("Ign 1: Voltage read: {}", voltage);
+
     return voltage; 
 }
 
-float Sensor::ReadIgnitorTwoContinuity(){
+float Sensor::ReadIgnitorTwoContinuity() {
     // voltage < 10 implies no continuity, and >= 10 is continuity
     float voltage = Read(IGN_2_CONT_CHANNEL, ADC2_INDEX);
+    spdlog::info("Ign 2: Voltage read: {}", voltage);
+
     return voltage; 
 }
```

Comment:
The thing about these functions is that they get called multiple times a second so will fill up the entire screen with info. Maybe make it debug level logs so they don't show up by default but still get logged if we need them (I believe info is the lowest level that shows up by default in the console)

Reply:
Good point, although I manually set the log level that shows up in the console to be warning or higher (`console_sink->set_level(spdlog::level::warn);`). I'll make them debug level anyway given how frequent/hopefully useless they are.